packages/infra/infra-core/src/lib/patterns/serviceCiConfig.ts

```diff
@@ -32,4 +45,38 @@ export class ServiceCiConfig extends Construct implements IServiceCiConfig {
 
     this.defaultCachePaths = ['~/.pnpm-store'];
   }
+
+  protected getWorkspaceSetupCommands(
+    ...pnpmWorkspaceFilters: PnpmWorkspaceFilters[]
+  ) {
+    const filters = pnpmWorkspaceFilters.map(
+      (workspace) => ` --filter=${workspace}`
+    );
+
+    return [
+      'go install github.com/segmentio/chamber/v2@latest',
+      'npm i -g pnpm@^8.6.1',
+      `pnpm install \
+                --include-workspace-root \
+                --frozen-lockfile`.concat(...filters),
+    ];
+  }
+
+  protected getECRLoginCommand() {
+    const awsRegion = Fn.ref('AWS::Region');
+    const awsAccountId = Fn.ref('AWS::AccountId');
+
+    return `aws ecr get-login-password --region ${awsRegion} \
+       | docker login --username AWS --password-stdin ${awsAccountId}.dkr.ecr.${awsRegion}.amazonaws.com`;
+  }
+
+  protected getAssumeRoleCommands() {
+    return [
+      'TEMP_ROLE=`aws sts assume-role --role-arn $ASSUME_ROLE_ARN --role-session-name test`',
```

Comment:
Is this `test` name intended to be like that or this is sth from testing?

Reply:
This change was introduced by @pziemkowski and the only thing I did here was to move it to the base serviceCiConfig to get rid of the code duplication in different CiConfigs